docs/output.md

```diff
@@ -0,0 +1,23 @@
+# Output Definitions
+This page aims to explain and define the various fields that will be present in the output.
```

Comment:
This is specific to CSV output, better to call it out explicitly

Reply:
What's different about this JSON and SARIF outputs?

Reply:
JSON has additional fields like metadata, risk uri. SARIF also has a subset of fields I think

Reply:
JSON has more fields. for SARIF the format is very SARIF specific, don't think we have to document it

Reply:
Ahh ok, CSV Output it is then haha